en_us.lang

```diff
@@ -281,83 +281,83 @@ gui.xaero_24h=24h
 gui.xaero_connect_with_auto_msg=Would you like to create the following connection between sub-worlds?
 gui.xaero_disconnect_from_auto_msg=Would you like to delete the following connection between sub-worlds?
-gui.xaero_teleport_not_connected=You are trying to teleport to a sub-world which isn't connected to the current "auto" one. If you are sure that this sub-world is from the same sub-server/world save, then you can enable teleportation by adding a connection in the waypoints menu -> Options -> Add Sub-World Connection. However, if you connect unrelated sub-worlds and teleport, then you might suffocate in a block or die from fall damage, so don't do that.
+gui.xaero_teleport_not_connected=You are trying to teleport to a sub-world that is not connected to the current auto one. If you are sure that this sub-world is from the same world save as the auto one, then you can enable teleportation by adding a connection on the waypoint screen -> Options -> Add Sub-World Connection. But beware, if you connect unrelated sub-worlds and teleport, then there is a good chance you will suffocate in a block or die from fall damage, so be careful and connect only the right ones.
 gui.xaero_ignore_heightmaps=Ignore Server Heightmaps
-gui.xaero_box_ignore_heightmaps=Some servers put custom values in the world heightmaps, which can cause incorrectly rendered maps. Sometimes it can even happen in heavily modded singleplayer worlds. This option should fix such issues at the expense of a bit of performance. Do not use unless you need to. Reenter the world after toggling the option!
+gui.xaero_box_ignore_heightmaps=Some servers put custom values in the world heightmaps, which can result in incorrectly rendered maps. Sometimes it can even happen in heavily modded singleplayer worlds. This option should fix such issues at the expense of a bit of performance. Do not use unless you have to. Reenter the world after toggling this option!
 gui.xaero_error_loading_properties=Error loading server minimap properties. Please retry.
 gui.xaero_light_block=Block
 gui.xaero_light_sky=Sky
 gui.xaero_light_both=All
-gui.xaero_waypoints_bottom=New Waypoints To Bottom
-gui.xaero_box_waypoints_bottom=Put newly created waypoints at the bottom of the waypoints set instead of the default top.
+gui.xaero_waypoints_bottom=New Waypoints to Bottom
+gui.xaero_box_waypoints_bottom=Put newly created waypoints at the bottom of the waypoint list instead of the default top.
 gui.xaero_armour_main_hand=Main Hand Item
-gui.xaero_armour_off_hand=Off-hand Item
+gui.xaero_armour_off_hand=Off-Hand Item
 gui.xaero_minimap_shape=Shape
 gui.xaero_minimap_shape_square=Square
 gui.xaero_minimap_shape_circle=Circle
 gui.xaero_slopes_legacy=Legacy
```

Comment:
Just a quick note here. "Legacy" in computing may also mean obsolete, outdated, stale, etc. If you don't plan to remove the old lighting (IMO the better one ATM) in the near future, what do you think about changing it to "classic" (like in Arrow Color: Classic and Choose a Preset -> Classic)?